conditions/conditions.go

```diff
@@ -0,0 +1,37 @@
+package conditions
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ConditionReason represents a condition reason.
```

Comment:
this comment doesn't help at all imo. Remove it or change it would be my suggestion

Reply:
Changed to:
```
// ConditionReason is a string representing a Kubernetes condition reason.
```